lib/mixlib/shellout/windows.rb

```diff
@@ -107,12 +107,19 @@ def run_command
                 # Kill the process
                 if (Time.now - start_wait) > timeout
                   begin
+                    require 'wmi-lite/wmi'
+                    wmi = WmiLite::Wmi.new
+                    Utils.kill_process_tree(process.process_id, wmi, logger)
```

Comment:
Add a message around here to the logger that's "louder," like what's in Mixlib::ShellOut::Unix:

```
logger.error("Command exceeded allowed execution time, sending KILL") if logger
```

That way something bubbles up to let us know something went wrong, in case something goes wrong while trying to kill the processes or we don't have logging high enough for the any of the debugging messages.